sdk/openai/Azure.AI.OpenAI/api/Azure.AI.OpenAI.netstandard2.0.cs

```diff
@@ -2,20 +2,39 @@ namespace Azure.AI.OpenAI
+        public static Azure.AI.OpenAI.ChatChoice ChatChoice(Azure.AI.OpenAI.ChatMessage message = null, int index = 0, Azure.AI.OpenAI.CompletionsFinishReason finishReason = default(Azure.AI.OpenAI.CompletionsFinishReason), Azure.AI.OpenAI.ChatMessage deltaMessage = null, Azure.AI.OpenAI.ContentFilterResults contentFilterResults = null) { throw null; }
+        public static Azure.AI.OpenAI.ChatCompletions ChatCompletions(string id = null, System.DateTimeOffset created = default(System.DateTimeOffset), System.Collections.Generic.IReadOnlyList<Azure.AI.OpenAI.ChatChoice> choices = null, System.Collections.Generic.IReadOnlyList<Azure.AI.OpenAI.PromptFilterResult> promptFilterResults = null, Azure.AI.OpenAI.CompletionsUsage usage = null) { throw null; }
+        public static Azure.AI.OpenAI.Choice Choice(string text = null, int index = 0, Azure.AI.OpenAI.ContentFilterResults contentFilterResults = null, Azure.AI.OpenAI.CompletionsLogProbabilityModel logProbabilityModel = null, Azure.AI.OpenAI.CompletionsFinishReason finishReason = default(Azure.AI.OpenAI.CompletionsFinishReason)) { throw null; }
+        public static Azure.AI.OpenAI.ContentFilterResult ContentFilterResult(Azure.AI.OpenAI.ContentFilterSeverity severity = default(Azure.AI.OpenAI.ContentFilterSeverity), bool isFiltered = false) { throw null; }
+        public static Azure.AI.OpenAI.ContentFilterResults ContentFilterResults(Azure.AI.OpenAI.ContentFilterResult sexualFilterResult = null, Azure.AI.OpenAI.ContentFilterResult violenceFilterResult = null, Azure.AI.OpenAI.ContentFilterResult hateFilterResult = null, Azure.AI.OpenAI.ContentFilterResult selfHarmFilterResult = null) { throw null; }
+        public static Azure.AI.OpenAI.ImageGenerations ImageGenerations(System.DateTimeOffset created, System.Collections.Generic.IEnumerable<Azure.AI.OpenAI.ImageLocation> data) { throw null; }
+        public static Azure.AI.OpenAI.ImageLocation ImageLocation(System.Uri url) { throw null; }
+        public static Azure.AI.OpenAI.PromptFilterResult PromptFilterResult(int index = 0, Azure.AI.OpenAI.ContentFilterResults contentFilterResults = null) { throw null; }
+        public static Azure.AI.OpenAI.StreamingChatChoice StreamingChatChoice(Azure.AI.OpenAI.ChatChoice originalBaseChoice = null) { throw null; }
+        public static Azure.AI.OpenAI.StreamingChatCompletions StreamingChatCompletions(Azure.AI.OpenAI.ChatCompletions baseChatCompletions = null, System.Collections.Generic.List<Azure.AI.OpenAI.StreamingChatChoice> streamingChatChoices = null) { throw null; }
+        public static Azure.AI.OpenAI.StreamingChoice StreamingChoice(Azure.AI.OpenAI.Choice originalBaseChoice = null) { throw null; }
+        public static Azure.AI.OpenAI.StreamingCompletions StreamingCompletions(Azure.AI.OpenAI.Completions baseCompletions = null, System.Collections.Generic.List<Azure.AI.OpenAI.StreamingChoice> streamingChoices = null) { throw null; }
```

Comment:
I was wondering how it is possible that there are some duplicates in the custom partial class versus the generated partial class without that being a build error. Turns out the classes have different names! 😱 The name of the generated one sounds wrong ("AIOpenAIModelFactory"). By any chance, do you know where that name came from? It might just be a bug in the generator. If there's nothing we can do, the best solution might be to rename the custom class to align with the generated one for now. ☹️

Reply:
Per the offline conversation, you're completely right and the name divergence is what caused all this duplication.

I commented everything out and then cross-referenced after merging the name (to the custom one, via `[CodeGenModel]`) to narrow the "custom" part down to the truly missing (for now) items.